server/config/environment.go

```diff
@@ -10,34 +10,35 @@ import (
 )
 
 type envVars struct {
-	SUBSCRIPTION               string
-	RESOURCE_GROUP_NAME        string
-	CONTAINER_GROUP_NAME       string
-	STORAGE_ACCOUNT_NAME       string
-	PASSWORD                   string
-	BASE_PATH                  string
-	DB_REL_PATH                string
-	TEMPLATE_REL_PATH          string
-	MAIN_OUTPUTS               string
-	ENGINE_END_WAIT            int64
-	ENGINE_MAX_RUNTIME         int64
-	ENGINE_RETRY_WAIT          int64
-	EXECUTION_MAX_RETRY        int
-	AZURE_POLLING_FREQ_SECONDS int
-	AUTO_RETRY                 bool
-	AUTO_RETRY_DELAY           int
-	SESSION_COOKIE_NAME        string
-	SESSION_COOKIE_PATH        string
-	SESSION_COOKIE_DOMAIN      string
-	SESSION_COOKIE_SECURE      bool
-	SESSION_COOKIE_MAX_AGE     int
-	SAVE_CONTAINER             bool
-	SEGMENT_WRITE_KEY          string
-	APPLICATION_ID             string
-	START_TIME                 string
-	LOG_REL_PATH               string
-	LOG_LEVEL                  string
-	AZURE_LOGIN_RETRIES        int
+	SUBSCRIPTION                 string
+	RESOURCE_GROUP_NAME          string
+	CONTAINER_GROUP_NAME         string
+	STORAGE_ACCOUNT_NAME         string
+	PASSWORD                     string
+	BASE_PATH                    string
+	DB_REL_PATH                  string
+	TEMPLATE_REL_PATH            string
+	MAIN_OUTPUTS                 string
+	ENGINE_END_WAIT              int64
+	ENGINE_MAX_RUNTIME           int64
+	ENGINE_RETRY_WAIT            int64
+	EXECUTION_MAX_RETRY          int
+	AZURE_POLLING_FREQ_SECONDS   int
+	AZURE_DEPLOYMENT_TIMEOUT_MIN int
```

Comment:
Can we rename this to add word "step" in there? `AZURE_DEPLOYMENT_STEP_TIMEOUT_MIN` or `DEPLOYMENT_STEP_TIMEOUT_MIN`

Reply:
Sure, done